packages/block-editor/src/components/block-list/insertion-point.js

```diff
@@ -84,6 +89,25 @@ export default function InsertionPoint( {
 		setInserterClientId( clientId );
 	}
 
+	function onClick( event ) {
+		const { clientX, clientY, target } = event;
+
+		if ( target !== ref.current ) {
```

Comment:
If I'm interpreting this correctly, this exists to ensure that the click has happened _specifically_ on the wrapper, and isn't bubbled from the inserter itself (where we'd definitely not want to be transitioning the caret)?

```suggestion
		// Only handle click on the wrapper specifically, and not an event
		// bubbled from the inserter itself.
		if ( target !== ref.current ) {
```

Reply:
Thanks :)